src/sonic-yang-models/yang-models/sonic-bgp-prefix-list.yang

```diff
@@ -0,0 +1,63 @@
+
+module sonic-bgp-prefix-list {
+
+    yang-version 1.1;
+
+    namespace "http://github.com/sonic-net/sonic-bgp-prefix-list";
+    prefix bgppl;
+
+    import sonic-types {
+        prefix stypes;
+    }
+
+    import sonic-extension {
+        prefix ext;
+    }
+
+    description "SONIC Device-specfifc BGP prefix lists data";
+
+    revision 2025-02-05 {
+        description "Initial revision.";
+    }
+
+    container sonic-bgp-prefix-list {
+
+        container PREFIX_LIST {
+
+            description "PREFIX_LIST part of config_db.json";
```

Comment:
change the description to "Different types of Prefix list entries consumed in BGP "